### DISCUSSION TITLE
General Talks

### DISCUSSION THREAD

**Discussion by data-with-joe** [commonwarexyz/monorepo#247] [General]
Hi @patrick-ogrady, gm!.
I don't think migrating to github is the best idea. Community members would hardly come here because they aren't familiar with github. Instead they prefer discord because it less technical and they are used to that as they can navigate multiple communities without switching apps.

**Comment by data-with-joe** [comment]
I understand the aim to move everything onchain, but github isn't onchain nor a web3 platform.

**Comment by patrick-ogrady** [comment]
Thanks for reaching out, @spec-gaming!

Our two primary focuses at the moment are growing the number of active contributors to the Commonware Library (this repository) and supporting great engineers building applications with it. As a small (but growing) team, we need to pick where to spend our time carefully. After the launch yesterday, we started spending a good amount of time in the Discord and most of those interactions ended up being with either bots, people trying to sell us something, or folks using our server as a place to advertise for something. We didn't have any interactions with engineers asking questions about the repository or looking for guidance on how to use the existing primitives we've already released.

With this all in mind, we decided to pause the Discord and go all-in on GitHub (and did so as soon it was clear it wasn't the right thing for us now to minimize confusion). As things evolve over the coming months, we may determine that having a strong Discord or Telegram presence would be beneficial and at that time would return properly staffed to support it (to do a great job rather than just enough to keep the lights on).

Look forward to seeing you around 🫡

**Comment by JackLamCHN** [comment]
The project is in different states, which leads to varying types of priorities. It is evident that Commonware is currently in its early development phase, where the priority is the highest. Compared to the Discord community aimed at the ecosystem and general users, GitHub Discussions can more efficiently meet the communication needs of developers and the project's priorities at this stage. Of course, as the project develops, I believe there will also be Discord and Telegram communities in the future; it’s just that the project has not reached that stage yet.

**Comment by data-with-joe** [comment]
All you mentioned makes complete sense. I'd try to contribute where I can.
Would be cool If you still post updates on the discord server tho :)

**Comment by mazedent2** [comment]
Hello please create a discord channel for community

**Reply by JackLamCHN** [reply]
Commonware has a Discord, but currently wants to focus on technical development, so using GitHub Discussions is more suitable for developer discussions and communication. Therefore, Discord is currently in a state where all channels are muted.
